src/lib/browserSaveFile.ts

```diff
@@ -15,7 +15,11 @@ export const browserSaveFile = async (blob: Blob, suggestedName: string) => {
       }
     })()
   // If the File System Access API is supported…
-  if (supportsFileSystemAccess && window.showSaveFilePicker) {
+  if (
+    supportsFileSystemAccess &&
+    window.showSaveFilePicker &&
+    !(window as any).playwrightSkipFilePicker
```

Comment:
This is a bit of a hack for sure, but I can't test the exports without skipping the file picker.